include/executor.hpp

```diff
@@ -313,6 +313,15 @@ class Executor
      */
     void sendFuncNumToPhyp(const types::FunctionNumber& funcNumber);
 
+    /**
+     * @brief API to display SRC functions 12 and 13
```

Comment:
This displays the hex words not the SRC function. either 2..5 or 6..9. :)

Reply:
changed the function name